.circleci/ansible/get_binaries.sh

```diff
@@ -16,7 +16,7 @@ if [ "${RELEASE}" = "true" ]; then
     fi
 else
     echo "search latest dev tag"
-    VERSION=$(curl -s https://registry.hub.docker.com/v1/repositories/neondatabase/neon/tags |jq -r -S '.[].name' | grep -v release | tail -1)
+    VERSION=$(curl -s https://registry.hub.docker.com/v1/repositories/neondatabase/neon/tags |jq -r -S '.[].name' | grep -v release | grep '^[0-9]*$' | sort -n | tail -1)
```

Comment:
nit: grep '^[0-9]+$' is a bit safer
nit: grep -v release is unecessary

Reply:
I've tested grep on MacOS and there grep hasn't `-E`, about `+` you are right but it should be escaped probably (in shell it should be `^[0-9]\+` but not sure how it should be inside docker in CircleCI

Reply:
Strange! My mac has `grep -E` but may be I've overwrote standard grep, not sure

Reply:
yes, seems my fault, updated script